book/src/move-basics/generics.md

```diff
@@ -22,20 +22,20 @@ enclosed in angle brackets (`<` and `>`). The generic parameters are separated b
 
 In the example above, `Container` is a generic type with a single type parameter `T`, the `value`
 field of the container stores the `T`. The `new` function is a generic function with a single type
-parameter `T`, and it returns a `Container` with the given value. Generic types must be initialed
+parameter `T`, and it returns a `Container` with the given value. Generic types must be initialized
 with a concrete type, and generic functions must be called with a concrete type.
 
 ```move
 {{#include ../../../packages/samples/sources/move-basics/generics.move:test_container}}
 ```
 
-In the test function `test_generic` we demonstrate three equivalent ways to create a new `Container`
-with a `u8` value. Because numeric types need to be inferred, we specify the type of the number
-literal.
+In the test function `test_generic`, we demonstrate three equivalent ways to create a new `Container`
+with a `u8` value. Since numeric types are inferred, we explicitly specify the type of the numeric
```

Comment:
```suggestion
with a `u8` value. Because numeric types are inferred, we explicitly specify the type of the numeric
```
Suggest keeping "because". Better choice for esl.